lib/flutter_map_tappable_polyline.dart

```diff
@@ -171,10 +176,15 @@ class TappablePolylineLayer extends StatelessWidget {
                         lengthDToOriginalSegment <
                             polylineOpts.pointerDistanceTolerance) {
                       onTap(currentPolyline);
+                      hit = true;
                       return;
                     }
                   }
                 });
+
+                if (!hit) {
```

Comment:
This breaks example. You should fix it

Reply:
What do you think about calling another function on miss, instead of calling the onTap method? E.g refactor so that we pass in two functions, "onTapHit" and another function "onTapMiss"?

Reply:
As we'll release v1 after this merge we can break things.

IMHO always testing if `polyline is TaggedPolyline` in the callback is a bit annoying for DX.
A second **optional** argument for misses looks good to me.